### ISSUE TITLE
Optimize Synced Tab for large dynamic type

### ISSUE THREAD

**Issue by data-sync-user** [mozilla-mobile/firefox-ios#11338]
When using Accessibility Dynamic Type sizes texts of the synced tab are not very well readable. For that reason we should layout the image, the tab label and the device name differently:

1. Image on top (left aligned)
  1. should be right aligned for RTL
1. Tab label underneath
1. Device name last

This layout change should only happen for Accessibility Dynamic Type sizes (UIContentSizeCategory.isAccessibilityCategory).

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FXIOS-4593)

**Comment by data-sync-user** [comment]
➤ Simion Basca commented:

Ignore my previous comment. This is fixed on latest main v9000 (19831) but it’s not uplifted to v106 (19787)

  1. Image on top (left aligned) ✅ 
    1. should be right aligned for RTL ✅ 

  2. Tab label underneath ✅ 
  3. Device name last ✅

**Comment by data-sync-user** [comment]
➤ Simion Basca commented:

Daniela Arcese should this  be uplifted this to v106.0? I am not seeing these changes in v106 (19846)

**Comment by data-sync-user** [comment]
➤ Daniela Arcese commented:

Simion Basca thanks for catching that, we uplifted it now.

**Comment by data-sync-user** [comment]
➤ Simion Basca commented:

Daniela Arcese please note that the layout seems broken for iPad in Portrait mode. See https://github.com/mozilla-mobile/firefox-ios/issues/12102 ( https://github.com/mozilla-mobile/firefox-ios/issues/12102|smart-link )

**Comment by data-sync-user** [comment]
➤ Simion Basca commented:

Marking as verified fix on v106.0 (20167)
